extensions/mite/src/time-entry.tsx

```diff
@@ -16,6 +16,11 @@ export default function Command() {
       });
       if (res.ok) {
         showToast({ title: "Entry created", style: Toast.Style.Success });
+        setTimeout(() => {
+          popToRoot({
+            clearSearchBar: false,
+          });
+        }, 1000);
```

Comment:
Is it necessary to use `setTimeout` here?

Reply:
hey @pernielsentikaer, I've removed this now in the latest commit. This was me following the UX flow found in `setup-fast-time-entry`. I've also removed the setTimeout from this too as I believe it's not necessary either.

Reply:
I've added the `setTimeout` to allow the user to see the form being submitted...i didn't like to be immediately moved outside without actually have any confirmation that the form was submitted.

Reply:
I think it makes sense to remove so we're using the same flow as the rest of Raycast 🙂